src/server/ApiRoutes/ResearchNow.js

```diff
@@ -0,0 +1,24 @@
+import nyplApiClient from '../routes/nyplApiClient';
+import {
+  createResearchNowQuery,
+} from '../../app/utils/utils';
+import appConfig from '../../app/data/appConfig';
+
+const appEnvironment = process.env.APP_ENV || 'production';
+
+const nyplApiClientCall = query => nyplApiClient({ apiBaseUrl: appConfig.api.drbb[appEnvironment] })
+  .then(client => client.get(`/research-now/v3/search-api?${query}`, { cache: false }))
+  .catch(console.error);
```

Comment:
If above change to appConfig is made, then you could pass in `apiName` to `nyplApiClient`, which is now acts as a kind of "api client builder", returning a different kind of client depending on what named API you specify:

```
const nyplApiClientCall = query => nyplApiClient({ apiName: 'drbb' })
  .then(client => client.get(`/research-now/v3/search-api?${query}`, { cache: false }))
```